__tests__/ResourceStateBuilder.test.js

```diff
@@ -16,8 +24,8 @@ describe('ResourceStateBuilder', () => {
     shortid.generate = jest.fn().mockReturnValueOnce('abc123').mockReturnValueOnce('def456').mockReturnValueOnce('ghi789')
 
     const builder = new ResourceStateBuilder(dataset, 'http://example/123')
-    const resourceState = builder.state
-    expect(resourceState).toEqual({
+    const result = await builder.buildState()
```

Comment:
Can you destructucture here? e.g. `const [state, leftovers, body] = ...`?

Reply:
Yes. Fixing here and elsewhere.

Reply:
Reason I wasn't destructuring was that I couldn't figure out how to handle the unused variables. However, since you pointed out `[state,]` (or similar), problem solved.